packages/@okta/vuepress-site/docs/release-notes/2025-okta-identity-engine/index.md

```diff
@@ -6,6 +6,148 @@ title: Okta Identity Engine API release notes 2025
+
+#### Developer documentation update in 2025.09.0
+
+* [Journeys](/docs/journeys/) is an evolving section that includes content to help guide you through your development project. Journeys break down your development project into consumable steps:
+
+  * Key concepts to absorb
+
+  * Information on planning for tasks that you need to complete
+
+  * Links to individual guides to help you build your project
+
+  * Additional content to help you enhance your project.
+
+  The next journey available is [Sign users in through your web app](/docs/journeys/OCI-web-sign-in/). This journey helps you connect your web app to Okta and sign your users in and out.
+
+* The new [Multifactor authentication](/docs/concepts/mfa/) concept page explains how MFA works and helps you understand some key terminology when you implement MFA using Okta APIs.
+
+* The new [OAuth 2.0 and OIDC claims](/docs/concepts/oauth-claims/) concept page provides a high-level overview of what claims are, including their types and how they're used in Okta.
+
+* The `revocation` and `revocationCacheLifetime` properties that are used with the Mutual TLS protocol in the [Identity Providers API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProvider/#tag/IdentityProvider/operation/createIdentityProvider!path=protocol/3/credentials/trust&t=request) are now deprecated. Okta now handles CRL caching automatically.
+
+    As of October 8, 2025, in Preview orgs, and October 13, 2025, in Production orgs, these properties are ignored if they’re specified in any API requests. Specifying the properties in your API requests doesn't cause any errors since they have no effect.
+
+    See [Deprecation Notice - Smart Card IdP Legacy CRL Cache Setting](https://support.okta.com/help/s/article/deprecation-notice-smart-card-idp-legacy-crl-cache-setting?language=en_US).
+
+* The new [Single sign-on](/docs/concepts/sso-overview/) concept is an overview of SSO, explaining what it is, how it differs from a basic sign-in flow, and how Okta supports it.
+
+* Authentication policies have been renamed and are now known as "app sign-in policies." The term "authentication policies" now refers to a group of policies: app sign-in policies, the Okta account management policy, and the session protection policy.
+
+    In the [Policies API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Policy/), the policy type for app sign-in policies is still `ACCESS_POLICY`. There are no changes to the API. All previous anchors and links to the API remain the same.
+
+* Best practice implementations of API use cases are now available for Identity Governance. See **Identity Governance** in the [Guides](/docs/guides/) section.
```

Comment:
```suggestion
* Best practice implementations of API use cases are now available for Identity Governance. See **Identity Governance** in the [Guides](/docs/guides/) navigation menu.
```

Reply:
Updated to sidebar in both classic and OIE